src/main/java/de/tum/in/www1/artemis/service/FeedbackService.java

```diff
@@ -37,4 +44,31 @@ public FeedbackService(FeedbackRepository feedbackRepository) {
         return feedbackRepository.findByReferenceInAndResult_Submission_Participation_Exercise(references, exercise).parallelStream()
                 .collect(toMap(Feedback::getReference, feedback -> feedback));
     }
+
+    // TODO: Move this to a StaticAssessmentService, implement it as a Helper Entity or directly in Feedback?
+    /**
+     * Transforms static assessment reports to feedback objects.
+     * As we reuse the Feedback entity to store static code analysis issues, a mapping to those attributes
+     * has to be defined, violating the first normal form.
+     *
+     * @param reports Static Assessment reports to be transformed
+     * @return Feedback objects storing the static assessment issues
+     */
+    public List<Feedback> createFeedbackFromStaticAssessmentReports(List<StaticAssessmentReportDTO> reports) {
+        List<Feedback> feedbackList = new ArrayList<>();
+        for (final var report : reports) {
+            StaticCodeAnalysisTool tool = report.getTool();
+
+            for (final var issue : report.getIssues()) {
+                Feedback feedback = new Feedback();
+                feedback.setText(tool.name());
```

Comment:
I suggest that we use a text identifier for static code analysis, e.g.
"StaticCodeAnalysisFeedback:<tool.name()>"